packages/fondue/src/components/Tooltip/Tooltip.tsx

```diff
@@ -37,21 +37,22 @@ const formatTooltipText = (text: string) => {
 };
 
 type TimeoutRef = {
-    current: NodeJS.Timeout | null;
+    current?: number;
```

Comment:
could you help me understand this change? Isn't the ref holding a reference to the timer?

Reply:
yes it is, why?